packages/types/lib/driver.ts

```diff
@@ -469,7 +469,17 @@ export interface ISessionHandler<
    * @returns A session data object
    */
   getSession(): Promise<SingularSessionData<C, SessionData>>;
-}
+
+  /**
+   * Return session capabilities info.
+   * FIXME: Please define better type after deciding the response format in
+   * https://github.com/appium/appium/issues/20880
+   * @param sessionId - the id of the session to return capabilities info.
+   * @returns A session capabilities object
+   */
+  getAppiumSessionCapabilities?(sessionId: string): Promise<{
```

Comment:
I still don't understand why we need a question mark after the method name

Reply:
Without `?`, driver build like fake driver with this type failed until such a driver implements `getAppiumSessionCapabilities`. I thought we won't block driver build with this update, so I added this `?`.

Btw, I was confused but I'm wondering if https://github.com/appium/appium/pull/20979 is a proper one at this time instead of this.